riker-macros/tests/macro.rs

```diff
@@ -38,77 +32,65 @@ impl Receive<String> for NewActor {
         println!("String");
     }
 }
-struct BasicActorRef;
-type Context<T> = Option<T>;
-
-trait Actor: Send + 'static {
```

Comment:
I'm confused
was this a test?
code duplication?
it's all gone now

Reply:
In order to test that the macro generates the correct code, the types used by the generated code must be in scope. In the old version, the code was duplicated. In the new version I added riker as a dev-dependecy so it that we can utilize the types directly inside the tests.

This reduces duplication and if the types in riker change but the macro isn't adapted to the change, the tests should fail and not silently keep working.

Reply:
i understood this the first explanation 😅but thank you for the summary 💜